exercises/chapter7/test/Main.purs

```diff
@@ -1,12 +1,207 @@
 module Test.Main where
 
 import Prelude
-
+import Data.AddressBook
+  ( PhoneType(..)
+  -- , address
+  -- , phoneNumber
+  )
+-- import Data.List
+--   ( List(..)
+--   , (:)
+--   )
+-- import Data.Maybe (Maybe(..))
+-- import Data.Traversable (traverse)
+-- import Data.Validation.Semigroup (invalid)
 import Effect (Effect)
-import Effect.Class.Console (logShow)
-
-import Data.AddressBook (examplePerson)
-import Data.AddressBook.Validation (validatePerson)
+-- import Test.Solutions
```

Comment:
I'm thinking it would be better to just let this be an implicit import (no parens, like `Prelude`), then you don't have to touch it when implementing the solutions.